packages/react-composites/review/react-composites.api.md

```diff
@@ -928,6 +929,12 @@ export type MessageReceivedListener = (event: {
 // @public
 export type MessageSentListener = MessageReceivedListener;
 
+// @beta
+export interface MobilePaneOptions {
+    // (undocumented)
+    disabledBackButton?: boolean;
```

Comment:
avoid boolean in the API interface as it becomes impossible to extend at a future date. Consider something like: 
`browserBackButtonOverride?: 'none' | 'closePane'`